### ISSUE TITLE
Simply GUI for better User Experience

### ISSUE THREAD

**Issue by mungewell** [gusmanb/PSVRFramework#37]
Obviously you and I are both techy geeks.... and love complicated interfaces ;-)
https://cloud.githubusercontent.com/assets/4086913/20158965/0d1e5242-a6de-11e6-92f6-ce3c7ab73b32.png

Unfortunately the rest of the populous is not the same. Put simply the interface is 'too much' and we should look to simplify as much as possible. Anything other than the basic functions should be hidden or under different sub-windows/menus.

Basic functions:
1). Enable VR mode, with streaming of UDP data (raw or processed)
2). Enable Cinematic mode, with setting of screen size/distance/brightness and recenter screen.
3). Power On/Off/Shutdown.

Simon

**Comment by gusmanb** [comment]
Agreed, the interface needs a revamp, maybe two modes, a basic one and an advanced one.

Will think about it next week, any suggestion or mock-up is welcome.

**Comment by gusmanb** [comment]
A thing I've been thinking and is related to this. I'm over the idea on sepparating the control from the UI.

The VR player now is using the UDP remote control to enter/exit from VR when a video starts/stops, so is very desirable to have always the control of the psvr started, also this can be some way of standarized acces to the device by third-party apps. I don't like startup programs, but this is very easily solved creating a service, just a service which listens and executes the commands, basically sepparate the UDP listener and the PSVR manipulation to a Windows service.

These services run perfectly on Linux and Mac, that's one thing I do each day on my work, so I have all the scripts for service control and can automate the service boot up even in Linux.

Also, other good thing about this is we can create different UI apps for different platforms, using WinForms for all the platforms feels alien on different platforms, that without counting the bugs it has, we can create a GTK# app for Linux and a Xamarin.Mac app for Mac (or even create these GUI's on some more "native" language/framework for each platform like Objective-C & Cocoa for Mac and C++ & Qt for Linux).

**Comment by mick-p1982** [comment]
Why are Cinema settings not saved across sessions? Am I missing something? Cinema is good for desktop use. Get the basics down pat.
